beacon-chain/core/blocks/block_operations.go

```diff
@@ -103,6 +103,15 @@ func verifySignature(signedData []byte, pub []byte, signature []byte, domain uin
 //    if state.eth1_data_votes.count(body.eth1_data) * 2 > SLOTS_PER_ETH1_VOTING_PERIOD:
 //        state.latest_eth1_data = body.eth1_data
 func ProcessEth1DataInBlock(beaconState *stateTrie.BeaconState, block *ethpb.BeaconBlock) (*stateTrie.BeaconState, error) {
+	if beaconState == nil {
+		return nil, errors.New("nil state")
+	}
+	if block == nil {
```

Comment:
wouldn't it be more efficient just do block == nil and block.Body == nil in one line?

Reply:
one line now. thanks